### DISCUSSION TITLE
升级到v5.1.0后traffic.sh运行不了了

### DISCUSSION THREAD

**Discussion by bobwang3** [v2fly/v2ray-core#2038] [Q&A]
traffic.sh是https://guide.v2fly.org/advanced/traffic.html 这里的。
现在运行会提示以下错误：

_./traffic.sh: line 11: /usr/local/bin/v2ctl: No such file or directory
------------Inbound----------
numfmt: input line is too short, no numbers found to convert in field 2
-------------User------------------------------------
numfmt: input line is too short, no numbers found to convert in field 2_

我发现v5.1.0安装好了之后v2ctl没有了。这个问题应该怎么解决？

**Comment by AkinoKaede** [comment]
Use `v2ray api`

**Reply by bobwang3** [reply]
> Use `v2ray api`

Can you be more specific?
